http-client/src/main/java/io/micronaut/http/client/netty/ConnectTTLHandler.java

```diff
@@ -30,7 +31,7 @@
  */
 public class ConnectTTLHandler extends ChannelDuplexHandler {
 
-    public static final AttributeKey<Boolean> RELEASE_CHANNEL = AttributeKey.newInstance("release_channel");
+    public static final AttributeKey<Boolean> CHANNEL_EXPIRED = AttributeKey.newInstance("channel_expired");
```

Comment:
this class is not `@Internal` so we can't replace this attribute, despite this being a more appropriate name. I'm not sure why japicmp didn't trigger though.